src/services/pin-state-transition/hooks/prepare-activity-log.js

```diff
@@ -54,8 +66,9 @@ const prepareActivityLog = () => (hook) => {
     const toBeNotifiedDepartments = [];
     let toBeNotifiedUsers = [];
     // Add current assigned_department/assigned_users to notification lists.
-    if (pin.assigned_department && pin.assigned_department._id) { // eslint-disable-line no-underscore-dangle,max-len
-      toBeNotifiedDepartments.push(pin.assigned_department._id); // eslint-disable-line no-underscore-dangle,max-len
+    if (pin.assigned_department) { // eslint-disable-line no-underscore-dangle,max-len
```

Comment:
Not allowed (https://github.com/supasate/youpin-api/blob/676f644d46ca088239ace6370819b294f49dfe3a/src/services/pin-state-transition/index.js#L34)